deploy/olm-catalog/application-monitoring-operator/monitoring-0.0.27/application-monitoring-operator.v0.0.27.clusterserviceversion.yaml

```diff
@@ -0,0 +1,332 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    categories: Monitoring stack
+    description: Operator managed and installs Prometheus, Alertmanager, Grafana
+    containerImage: quay.io/integreatly/application-monitoring-operator:0.0.27
+    createdAt: 2019-09-26 10:51:00 
+    support: Red Hat, Inc.
+    certified: "false"
+  name: application-monitoring-operator.v0.0.27
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+      - name: applicationmonitorings.applicationmonitoring.integreatly.org
+        version: v1alpha1
+        kind: ApplicationMonitoring
+        displayName: Integreatly monitoring stack.
+        description: Represents an installation of the Integreatly monitoring stack.
+      - name: blackboxtargets.applicationmonitoring.integreatly.org
+        version: v1alpha1
+        kind: BlackboxTarget
+        displayName: Integreatly monitoring blackbox target.
+        description: Represents a group of blackbox targets which prometheus blackbox exporter in the Integreatly monitoring stack will query.
+      - name: grafanas.integreatly.org
+        version: v1alpha1
+        kind: Grafana
+        displayName: Integreatly monitoring Grafana Operator.
+        description: Represents an installation of the Integreatly Grafana operator.
+      - name: grafanadashboards.integreatly.org
+        version: v1alpha1
+        kind: GrafanaDashboard
+        displayName: Integreatly monitoring Grafana dashboard.
+        description: Represents an instance of the Integreatly Grafana dashboard.
+      - name: grafanadatasources.integreatly.org
+        version: v1alpha1
+        kind: GrafanaDataSource
+        displayName: Integreatly monitoring Grafana datasource.
+        description: Represents an instance of the Integreatly Grafana datasource.
```

Comment:
@pb82 @david-martin this is the main change here, the AMO is currently responsible for getting these Grafana CRDs into the cluster, however we could/should maybe look at decoupling the grafana operator and use its own CSV (https://github.com/operator-framework/community-operators/tree/master/community-operators/grafana-operator), create a subscription and then create its CR via the AMO.

Reply:
Perhaps do the same for the Prometheus/Alertmanager operator too maybe?